usl_pipeline/cloud_functions/main.py

```diff
@@ -563,6 +564,16 @@ def _process_wps_feature(feature: xarray.DataArray, var_config: dict) -> NDArray
     if "num_metgrid_levels" in feature.dims:
         feature = feature.isel(num_metgrid_levels=0)
 
+    # Monthly climatology var - extract to only the month of the file's datestamp
+    if "z-dimension0012" in feature.dims and snapshot_time:
+        snapshot_datetime = datetime.datetime.strptime(
+            snapshot_time.values[0].decode(), "%Y-%m-%d_%H:%M:%S"
```

Comment:
Do we have a consistent name to get this value from? `snapshot_time.values[0]` is going to be grabbing an item at random. Is it always going to be a single value? Is that something we should assert?

Reply:
yeah this is always going to be a single valued array since each wps file corresponds to only one time value